src/main/java/frc/robot/Robot.java

```diff
@@ -16,8 +22,13 @@
  */
 public class Robot extends TimedRobot {
   private Command m_autonomousCommand;
-
+  private Drive drive;
+  private Gyroscope gyro;
   private RobotContainer m_robotContainer;
+  private Controls driverControls;
```

Comment:
Add a `OperatorInputs` object here and "wire" it up in the `robotInit()` method like we've done for the `drive`, `gyro`, and `driverControls` objects. Let me know if this is confusing and we will work through it